Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM ruby:2.1
+FROM node:stretch
```

Comment:
I would probably have chosen `debian:stretch` as we use that for almost all other images, but hey :man_shrugging: :slightly_smiling_face:

Reply:
Let me continue the nagging by saying "do'h" to the branch name :smile: 

```
$ git fetch origin ruby2.1->node.stretch
fatal: Couldn't find remote ref ruby2.1-
```

Reply:
Fair... Bullet 2... Difficult/annoying/time-consuming to install `node` in `debian:stretch`.